src/charts/axis.js

```diff
@@ -93,7 +93,7 @@ export default function Axis (_container) {
       cache.y2AxisRoot.select(".axis-group").append("g").attr("class", "axis y2")
     }
 
-    cache.xLabelsShouldRotate = shouldXLabelsRotate()
+    cache.xLabelsShouldRotate = config.labelsAreRotated === "auto" ? shouldXLabelsRotate() : config.labelsAreRotated
```

Comment:
This `=== "auto"` checking is duplicated over this and the logic below - an opportunity to clean this up some by moving that into `shouldXLabelsRotate` instead (so that the conditional below could just be the one function call). Otherwise, looks good for the current code.

Reply:
good call!